vignettes/GeomxTools_RNA-NGS_Analysis.Rmd

```diff
@@ -101,10 +101,15 @@ Our specific objectives:
 Let's install and load the GeoMx packages we need:
 
 ```{r InstallGeomxTools, echo = TRUE, eval = FALSE}
-install.packages("devtools")
-devtools::install_github("Nanostring-Biostats/NanoStringNCTools")
-devtools::install_github("Nanostring-Biostats/GeomxTools", ref = "dev")
-devtools::install_github("Nanostring-Biostats/GeoMxWorkflows", ref = "main")
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+# The following initializes most up to date version of Bioc
+BiocManager::install(version=3.15)
```

Comment:
I believe you should be able to do release. Here I believe instead of having to specify.

Reply:
I think you have to specify which release. I will add "" around the version number.
![image](https://user-images.githubusercontent.com/40255151/163846733-4c9be0f1-c572-4016-842d-5e08672412a7.png)